content/webapp/pages/article-series.tsx

```diff
@@ -147,8 +162,25 @@ const ArticleSeriesPage: FC<Props> = props => {
         contributors={series.contributors}
         seasons={series.seasons}
       >
-        {articles.length > 0 && (
-          <SearchResults items={series.items} showPosition={true} />
+        <SearchResults items={series.items} showPosition={true} />
+        {articles.totalPages > 1 && (
+          <Space
+            v={{ size: 'm', properties: ['padding-top', 'padding-bottom'] }}
+          >
+            <div className="text-align-right">
```

Comment:
nit; I think this class could be moved to the parent component and still work nicely? (Or made into a styled-component if it does)

Reply:
Good shout; I did this everywhere and was able to bin the `text-align-right` class (only used for these pagination buttons).